doc/lua_api.txt

```diff
@@ -2163,32 +2163,48 @@ For the following functions `x` can be either a vector or a number:
 
 Helper functions
 ----------------
-* `dump2(obj, name="_", dumped={})`
-     * Return object serialized as a string, handles reference loops
-* `dump(obj, dumped={})`
-    * Return object serialized as a string
+* `dump2(obj, name, dumped)`: returns a string which makes `obj` human readable,
+        handles reference loops
+    * `obj`: arbitrary variable
```

Comment:
Not an arbitrary variable, this is the object name surely?

Reply:
Maybe 'object name' or 'object ref'?

Reply:
I don't understand your question.

AfaIk dump is not a function for serialization because there's minetest.serialize
The obj parameter is not a minetest object but it's a function, string, table, nil, boolean, number or userdata.

Reply:
I was wrong http://irc.minetest.net/minetest-dev/2017-09-28#i_5091452

Reply:
Passing something which is not a variable only means passing the thing directly in lua I think. And passing something directly doesn't make sense, why would anyone do it?